skeleton/aws/modules/vpc/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "namespace" {
+}
+
+variable "cidr" {
+  description = "VPC CIDR"
+  default     = "10.0.0.0/16"
+}
+
+variable "private_subnet_cidrs" {
+  type    = list(any)
+  default = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+}
+
+variable "public_subnet_cidrs" {
+  type    = list(any)
```

Comment:
```suggestion
variable "public_subnet_cidrs" {
  description = "VPC public subnet CIDRs"
  type    = list(any)
```

Reply:
Added in https://github.com/nimblehq/infrastructure-templates/pull/27/commits/fe3318a48516b380ea6b34a5347ede9d6a35cfa0